pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -4381,3 +4381,70 @@ func TestMergeDBDeployment(t *testing.T) {
 		})
 	}
 }
+
+func TestJvmDefaultConf(t *testing.T) {
+	name := "test"
+	envReplace := corev1.EnvVar{
+		Name:  "KIE_ADMIN_PWD",
+		Value: "RedHat",
+	}
+	envAddition := corev1.EnvVar{
+		Name:  "CONSOLE_TEST",
+		Value: "test",
+	}
+	cr := &api.KieApp{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: api.KieAppSpec{
+			Environment: api.RhdmTrial,
+			CommonConfig: api.CommonConfig{
+				ApplicationName: "trial",
```

Comment:
this commonconfig section is probably unecessary for a jvm specific test.

Reply:
Ok